Userland/Libraries/LibHTTP/HttpResponse.cpp

```diff
@@ -18,4 +18,63 @@ HttpResponse::~HttpResponse()
 {
 }
 
+StringView HttpResponse::reason_phrase_for_code(int code)
+{
+    VERIFY(code > 0);
+
+    static HashMap<int, String> s_reason_phrases = {
+        { 100, "Continue"sv },
+        { 101, "Switching Protocols"sv },
+        { 200, "OK"sv },
+        { 201, "Created"sv },
+        { 202, "Accepted"sv },
+        { 203, "Non-Authoritative Information"sv },
+        { 204, "No Content"sv },
+        { 205, "Reset Content"sv },
+        { 206, "Partial Content"sv },
+        { 300, "Multiple Choices"sv },
+        { 301, "Moved Permanently"sv },
+        { 302, "Found"sv },
+        { 303, "See Other"sv },
+        { 304, "Not Modified"sv },
+        { 305, "Use Proxy"sv },
+        { 307, "Temporary Redirect"sv },
+        { 400, "Bad Request"sv },
+        { 401, "Unauthorized"sv },
+        { 402, "Payment Required"sv },
+        { 403, "Forbidden"sv },
+        { 404, "Not Found"sv },
+        { 405, "Method Not Allowed"sv },
+        { 406, "Not Acceptable"sv },
+        { 407, "Proxy Authentication Required"sv },
+        { 408, "Request Timeout"sv },
+        { 409, "Conflict"sv },
+        { 410, "Gone"sv },
+        { 411, "Length Required"sv },
+        { 412, "Precondition Failed"sv },
+        { 413, "Payload Too Large"sv },
+        { 414, "URI Too Long"sv },
+        { 415, "Unsupported Media Type"sv },
+        { 416, "Range Not Satisfiable"sv },
+        { 417, "Expectation Failed"sv },
```

Comment:
You cant add status codes, and not include `418 I'm a teapot` :^)

Reply:
It's sadly not part of the RFC, and I decided to stay with that list for the moment. :^)

Reply:
Thank you for not memeing in the code <3

Reply:
> Thank you for not memeing in the code <3

Although it is actually listed by [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses).